src/scripts/main.js

```diff
@@ -1,3 +1,36 @@
 'use strict';
 
-// write code here
+const container = document.querySelector('.container');
+const tbody = document.querySelector('table tbody');
+
+const appendRow = document.querySelector('.append-row');
+const removeRow = document.querySelector('.remove-row');
+const appendColumn = document.querySelector('.append-column');
+const removeColumn = document.querySelector('.remove-column');
+
+container.addEventListener('click', () => {
+  appendRow.disabled = tbody.children.length >= 10;
+  removeRow.disabled = tbody.children.length <= 2;
+  appendColumn.disabled = tbody.firstElementChild.children.length >= 10;
+  removeColumn.disabled = tbody.firstElementChild.children.length <= 2;
```

Comment:
NI
better to make constants for such `magic numbers`
const MAX_TABLE_SIZE = 10;
const MIN_TABLE_SIZE = 2;